apps/server/src/modules/tool/service/tool-validation.service.spec.ts

```diff
@@ -105,6 +105,16 @@ describe('ToolValidation', () => {
 				await expect(result).resolves.not.toThrow();
 			});
 
+			it('should validate the default with the regular expression', async () => {
```

Comment:
I dont understand these tests and i am missing the exception case.
Please prepare understandable data for this test